doc/orchestrator.rst

```diff
@@ -165,3 +165,287 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+example, we specify to SmartSim that we intend to execute the `Model` once on a single node.
+
+.. code-block:: python
+  srun_settings.set_nodes(1)
+  srun_settings.set_tasks_per_node(1)
+
+Next, create an instance of the Model object using the create_model() function of the
+exp object. Pass in the srun_settings object, which was configured earlier, as an argument
+to the create_model() function and assign to the variable ‘model’.
+
+.. code-block:: python
+  model = exp.create_model("tutorial-model", srun_settings)
+
+Next, launch the model instance using the exp.start() function.
+
+.. code-block:: python
+  exp.start(model, block=True, summary=True)
+
+Next in the example, add SmartRedis multi-database functionality to the Model.
+Navigate to the file specified in the exe_args argument of the create_run_settings()
+function, exe_args="/path/to/model_multidb_example.py".
+
+To begin, learn to retrieve data from the launched databases by using a SmartRedis Client
+and the database identifier.In the Model script, we do not have access to the
+Experiment.get_address() function. To establish a connection with the launched databases,
+we still need to initialize a SmartRedis client for each Orchestrator with the address of
+the launched database.
+
+Previously, we added the db_identifier argument when creating the two databases.
+When a database is created with a db_identifier name, SmartSim creates an environment
+variable named SSDB for each database with the db_identifier variable value suffixed to
+SSDB. In the case of the example, the environment variables created are SSDB_std-deployment
+and SSDB_clus-deployment.
+
+To create a SmartRedis Client for each database, first use the
+ConfigOptions.create_from_environment() factory method, which accepts the
+suffix to be applied to the SSDB environment variable. Then pass the ConfigOptions
+object when creating the SmartRedis client.
+
+For the standalone database:
+
+.. code-block:: python
+  from smartredis import ConfigOptions, Client
+  import torch
+
+  std_config = ConfigOptions.create_from_environment("std-deployment")
+  std_db_client = Client(std_config, "client_1")
+
+For the clustered database:
+
+.. code-block:: python
+  clus_config = ConfigOptions.create_from_environment("clus-deployment")
+  clus_db_client = Client(clus_config, "client_2")
+
+These steps allow us to set up SmartRedis clients for each Orchestrator and
+establish effective communication with the respective databases.
+
+To confirm a successful connection to each database, retrieve the tensors previously
+stored using the created SmartRedis clients. The `get_tensor` Client method allows
+retrieval of a tensor by passing in the tensor name.
+
+.. code-block:: python
+  val1 = std_db_client.get_tensor("tensor1")
+  val2 = clus_db_client.get_tensor("tensor2")
+  print(f"{val1} + {val2}")
+
+This code showcases that we can retrieve data from each database effectively.
```

Comment:
It would be better if we didn't put the same data into each DB because then it would be clear that we got the data back that we put in

Reply:
Can we show output for these commands?